packages/web/src/store/configureStore.ts

```diff
@@ -137,10 +136,7 @@ const syncClientStateToNativeMobile = (store: Store) => {
             previousState[k as keyof RootState]
         )
       ) {
-        // Debounce messages to minimize expensive stringify and parse.
-        // Leading and trailing states are sent, state will be out of sync
-        // a maximum of 500ms
-        postMessageDebounced({
+        postMessage({
```

Comment:
This was done initially to reduce message passing and rerenders under the assumption that serializing and parsing the state would be slow. But I noticed lag when updating the filters because multiple state updates would happen at once and the filter display wouldn't update for 500ms

From what I could tell this didn't cause a negative perf impact anywhere else, but let's keep an eye on it